docs/SDL/OnAppPermissionChanged/index.md

```diff
@@ -9,6 +9,28 @@ Sender
 Purpose
 : Inform HMI that permissions have changed for a specified application.
 
+Notification from SDL to HMI occurs when application permissions were changed. If no permission specified means that application was dissallowed and has to be unregistered.
+
+!!! MUST  
+1.  Request the list of permissions per app for which the consent is required (via `GetListOfPermissions`) when HMI gets the 'user-consent-required' notification per app from `SDL (SDL.OnAppPermissionChanged{appID, appPermissionsConsentNeeded: true})`.  
+2.  Request Policies Manager for the appropriate user-friendly message from Local PT (`SDL.GetUserFriendlyMessage`) to commuticate about permissions changed when HMI gets the list of permissions that require User's consent.
+
+_It is app’s responsibility to inform the user to change the app’s settings if user consent is needed for proper app operation.  
+Any permissions that require user consent are not enabled until a user accepts them. If permissions do not require user consent, they can be applied immediately._ 
+!!!
+
+_**SDL Note**:_   
+
+1. SDL sends the list of RequestTypes allowed by Policies via `OnAppPermissionChanged` API. In case HMI will use any of requestTypes disallowed by PolicyTable, SDL will ignore such notifications.  
+2. SDL PoliciesManager must send `OnAppPermissionChanged (appRevoked: true, appID)`and `BC.ActivateApp` (NONE) to HMI, in case the `<appID>` application is currently registered and in any HMILevel and in result of PTU `<appID>` gets "null" policies.   
+3. If app permissions were reduced after PTU, SDL sends `OnAppPermissionChanged (isAppPermissionsRevoked: true)` when app is in FULL or LIMITTED. SDL doesn't notify HMI about revoked permissions twice.  
+4. In case the Local PT is successfully updated AND there are new application permissions that require User's consent, Policies Manager must:  
+  * notify HMI about 'user-consent-required' via `SDL.OnAppPermissionChanged{appID, appPermissionsConsentNeeded: true}` per each application in FULL/LIMITED that lacks the User's permissions right after Policies Manager detects the user-unconsented permissions in Local PT  
+  * notify HMI about 'user-consent-required' `SDL.OnAppPermissionChanged{appID, appPermissionsConsentNeeded: true} after the related app is activated on HMI (in case the related app is now in BACKGROUND /NONE HMILevel)`.  
```

Comment:
Sadly, they really do need to be indented by 4 spaces, otherwise the dev portal just put all of these items on the same level for some reason.